lupa/tests/test.py

```diff
@@ -3030,7 +3030,7 @@ def test_decrease_memory(self):
         self.lua.set_max_memory(10000)
         self.assertGreater(self.lua.max_memory, 10000)
         self.assertGreaterEqual(self.lua.memory_used, 50000)
-        self.assertRaises(self.lupa.LuaMemoryError, self.lua.eval, "('a'):rep(10)")
+        self.assertRaises(self.lupa.LuaMemoryError, self.lua.eval, "('b'):rep(10)")
```

Comment:
Honestly no idea why this keeps failing sometimes [on windows](https://github.com/scoder/lupa/runs/7715974364?check_suite_focus=true).

Might be string reuse (changed to `b` to check) or maybe it puts small values on stack and not heap which doesn't call our custom allocator?